.github/compatibility-test-matrices/release-v7.2.x/client-chain-b.json

```diff
@@ -1,16 +1,15 @@
 {
   "chain-a": [
-    "release-v7.2.x",
-    "v7.1.0",
-    "v7.0.1",
+    "v7.2.0",
     "v6.2.0",
     "v6.1.1",
+    "v5.3.1",
     "v5.2.1",
     "v4.4.2",
     "v4.3.1",
     "v4.2.2",
     "v4.1.3",
-    "v5.3.1"
+    "release-v7.2.x"
```

Comment:
aha, we might also want this to be first

Reply:
Either top or bottom works for me. :)